api/csi.go

```diff
@@ -101,6 +111,19 @@ type CSIVolume struct {
 
 	CreateIndex uint64
 	ModifyIndex uint64
+
+	// ExtraKeysHCL is used by the hcl parser to report unexpected keys
+	ExtraKeysHCL []string `hcl:",unusedKeys" json:"-"`
+}
+
+// allocs is called after volume creation to collapse allocations for the UI
```

Comment:
Isn't this called after we query volumes, not after we create them? (i.e. it's like the denormalization we do in the state store)

Reply:
Myopia! it's called after the api.CSIVolume is created, i.e. after we query it.

Reply:
fixed the comments in 34dab4a3b